src/lib/Libattr/master_svr_attr_def.xml

```diff
@@ -92,1715 +92,1717 @@
+	<member_name>ATTR_max_run_soft</member_name>		<!-- "max_run_soft" -->
 	<member_at_decode>decode_entlim</member_at_decode>
 	<member_at_encode>encode_entlim</member_at_encode>
 	<member_at_set>set_entlim</member_at_set>
 	<member_at_comp>comp_str</member_at_comp>
 	<member_at_free>free_entlim</member_at_free>
 	<member_at_action>action_entlim_chk</member_at_action>
-	<member_at_flags><both>NO_USER_SET</both></member_at_flags>
-	<member_at_type><both>ATR_TYPE_ENTITY</both></member_at_type>
+	<member_at_flags>NO_USER_SET</member_at_flags>
+	<member_at_type>ATR_TYPE_ENTITY</member_at_type>
 	<member_at_parent>PARENT_TYPE_SERVER</member_at_parent>
 	<member_verify_function>
 	<ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
    <attributes>
-   /* SVR_ATR_max_run_res_soft */
-	<member_name><both>ATTR_max_run_res_soft</both></member_name>		<!-- "max_run_res_soft" -->
+   <member_index>SRV_ATR_max_run_res_soft</member_index>
+	<member_name>ATTR_max_run_res_soft</member_name>		<!-- "max_run_res_soft" -->
 	<member_at_decode>decode_entlim_res</member_at_decode>
 	<member_at_encode>encode_entlim</member_at_encode>
 	<member_at_set>set_entlim_res</member_at_set>
 	<member_at_comp>comp_str</member_at_comp>
 	<member_at_free>free_entlim</member_at_free>
 	<member_at_action>action_entlim_chk</member_at_action>
-	<member_at_flags><both>NO_USER_SET</both></member_at_flags>
-	<member_at_type><both>ATR_TYPE_ENTITY</both></member_at_type>
+	<member_at_flags>NO_USER_SET</member_at_flags>
+	<member_at_type>ATR_TYPE_ENTITY</member_at_type>
 	<member_at_parent>PARENT_TYPE_SERVER</member_at_parent>
 	<member_verify_function>
 	<ECL>NULL_VERIFY_DATATYPE_FUNC</ECL>
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
    <attributes>
-   /* SRV_ATR_MaxUserRun */
-	<member_name><both>ATTR_maxuserrun</both></member_name>	<!-- "max_user_run" -->
+    <member_index>SRV_ATR_MaxUserRun</member_index>
+	<member_name>ATTR_maxuserrun</member_name>	<!-- "max_user_run" -->
 	<member_at_decode>decode_l</member_at_decode>
 	<member_at_encode>encode_l</member_at_encode>
 	<member_at_set>set_l</member_at_set>
 	<member_at_comp>comp_l</member_at_comp>
 	<member_at_free>free_null</member_at_free>
 	<member_at_action>check_no_entlim</member_at_action>
-	<member_at_flags><both>NO_USER_SET</both></member_at_flags>
-	<member_at_type><both>ATR_TYPE_LONG</both></member_at_type>
+	<member_at_flags>NO_USER_SET</member_at_flags>
+	<member_at_type>ATR_TYPE_LONG</member_at_type>
 	<member_at_parent>PARENT_TYPE_SERVER</member_at_parent>
 	<member_verify_function>
 	<ECL>verify_datatype_long</ECL>
 	<ECL>NULL_VERIFY_VALUE_FUNC</ECL>
 	</member_verify_function>
    </attributes>
    <attributes>
-   /* SRV_ATR_MaxGrpRun */
-	<member_name><both>ATTR_maxgrprun</both></member_name>		<!-- "max_group_run" -->
+    <member_index>SRV_ATR_MaxGrpRun</member_index>
```

Comment:
alignment

Reply:
aligned all the xmls using a xml formatter online